x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_status.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { EcsEventOutcome } from 'kibana/server';
+
+interface RuleEventLogListStatusProps {
+  status: EcsEventOutcome;
+}
+
+const statusContainerStyles = {
+  display: 'flex',
+  alignItems: 'center',
+  textTransform: 'capitalize' as const,
+};
+
+const iconStyles = {
+  marginRight: '8px',
+};
+
+const STATUS_TO_COLOR: Record<EcsEventOutcome, string> = {
+  success: 'success',
+  failure: 'danger',
+  unknown: 'gray',
+};
+
+export const RuleEventLogListStatus = (props: RuleEventLogListStatusProps) => {
+  const { status } = props;
+  return (
+    <div style={statusContainerStyles}>
+      <EuiIcon type="dot" color={STATUS_TO_COLOR[status]} style={iconStyles} />
```

Comment:
Hmmm we don't really have a fallback state design for the statuses, we could either render nothing, or still render it but with a default dot color? like gray or something?

since the status is just a string that is arbitrarily used to filter the log result, it might ok to just have a default dot color?